src/main/java/lotto/Lotto.java

```diff
@@ -0,0 +1,73 @@
+package lotto;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import lotto.model.LottoNumber;
+import lotto.model.LottoNumbers;
+
+public class Lotto {
+	private static Scanner scan = new Scanner(System.in);
+	public static void main(String[] args) {
+		String money = inputMessage("구입금액을 입력해 주세요.");
+		
+		LottoMachine lottoMachine = new LottoMachine(money);
+		buyAutoLotto(lottoMachine.buyAutoLottos());
+
+		String lastLottoNumbers = inputMessage("\n지난 주 당첨 번호를 입력해 주세요.");
+		List<LottoNumber> lottoNumbers = lastWinLottoNumbers(lastLottoNumbers);
+		
+		printWinStatistics(lottoMachine.winList(new LottoNumbers(lottoNumbers)));
+		System.out.println(profitRateStringFormat(lottoMachine.profitRate()));
+	}
+	
+	private static String inputMessage(String message) {
+		System.out.println(message);
+		return scan.nextLine();
+	}
+	
+	private static void buyAutoLotto(List<LottoNumbers> lottos) {
+		System.out.println(lottoNumbersTitleStringFormat(lottos));
+		for (int i=0; i<lottos.size(); ++i) {
+			System.out.println(lottoNumbersStringFormat(lottos.get(i)));
+		}
+	}
+	
+	private static List<LottoNumber> lastWinLottoNumbers(String lastLottoNumbers) {
+		List<LottoNumber> lottoNumbers = new ArrayList<LottoNumber>();
```

Comment:
타입 아규먼트는 생략할 수 있습니다.
```suggestion
		List<LottoNumber> lottoNumbers = new ArrayList<>();
```